static/index.html

```diff
@@ -24,6 +24,10 @@ <h1><img src="static/images/c4sj_logo.jpg" alt="Code For San Jose" id="c4sj_logo
       <label>
         <span>Name</span>
         <input type="text" name="name" placeholder="your name" required autofocus>
+      </label>
+       <label>
+        <span>Preferred Pronoun</span>
+        <input type="text" name="pronoun" placeholder="your preferred pronoun" required autofocus>
```

Comment:
Not everyone would know what to put here if it's a blank box (I'd never even heard of the term "preferred pronoun" until 2 years ago). Can we make this be a radio button or dropdown with some presets?

Here are some suggestions:
* he, him, his
* she, her, hers
* they
* ze, hir, hirs
* other (if this is selected, a text box will appear to allow any entry)